mods/tnt/init.lua

```diff
@@ -1,10 +1,10 @@
 tnt = {}
 -- Default to enabled in singleplayer and disabled in multiplayer
 local singleplayer = minetest.is_singleplayer()
-local setting = minetest.setting_getbool("enable_tnt")
-if (not singleplayer and setting ~= true) or
-		(singleplayer and setting == false) then
-	return
+local enable_tnt = minetest.setting_getbool("enable_tnt")
+if (not singleplayer and enable_tnt ~= true) or
+		(singleplayer and enable_tnt == false) then
+	enable_tnt = false
```

Comment:
Ah, that originally `return`ed from the file. Yea, then the if-statement can go completly.